app/service/search_index.rb

```diff
@@ -0,0 +1,17 @@
+class SearchIndex
+  def self.add(record)
+    if search_index_callbacks_enabled?
+      IndexerJob.perform_async(:index, record.id)
+    end
+  end
+
+  def self.remove(record)
+    if search_index_callbacks_enabled?
+      IndexerJob.perform_async(:delete, record.id)
+    end
+  end
+
+  def self.search_index_callbacks_enabled?
+    ENV.fetch('ENABLE_SEARCH_INDEX_CALLBACKS', true) != 'false'
```

Comment:
Actually, on second thought, this probably doesn't need to be added to `.env.sample`. Since it defaults to true I think we're good.